cli/openapi/api_api.go

```diff
@@ -47,8 +47,8 @@ CreateEnvironment Create new environment
 
 Create new environment action
 
- @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
- @return ApiCreateEnvironmentRequest
+	@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
```

Comment:
@xoscar how are you generating this files? we shouldn't have spacing diffs here

Reply:
That was strange. At first, I thought it was a versioning issue, but we are pinning the generator version in the makefile.